include/net/if6.h

```diff
@@ -49,20 +56,25 @@ struct in6_aliasreq {
 	struct in6_addrlifetime ifra_lifetime;
 };
 
+
+/* lwIP configuration */
+#define LWIP_IPV6_NUM_ADDRESSES 3
+
 /* IPv6 ioctls */
-#define SIOCGIFDSTADDR_IN6     _IOWR(SOCK_IOC_TYPE, 0x22, struct in6_ifreq)
-#define SIOCGIFNETMASK_IN6     _IOWR(SOCK_IOC_TYPE, 0x25, struct in6_ifreq)
-#define SIOCDIFADDR_IN6        _IOW(SOCK_IOC_TYPE, 0x19, struct in6_ifreq)
-#define SIOCAIFADDR_IN6        _IOW(SOCK_IOC_TYPE, 0x1a, struct in6_aliasreq)
-#define SIOCGIFAFLAG_IN6       _IOWR(SOCK_IOC_TYPE, 0x49, struct in6_ifreq)
-#define SIOCGIFALIFETIME_IN6   _IOWR(SOCK_IOC_TYPE, 0x51, struct in6_ifreq)
+#define SIOCGIFADDR_IN6      _IOWR(SOCK_IOC_TYPE, 0x21, struct in6_ifreq[LWIP_IPV6_NUM_ADDRESSES])
```

Comment:
Using `LWIP_IPV6_NUM_ADDRESSES` introduces an unwanted dependency on lwIP for libphoenix.
If you prefer to use `SIOCGIFNETMASK_IN6` instead of `getifaddrs` to retrieve IPv6 addresses, please design an API that avoids introducing this dependency.